Formula/c/context7-mcp.rb

```diff
@@ -0,0 +1,26 @@
+class Context7Mcp < Formula
+  desc "Up-to-date code documentation for LLMs and AI code editors"
+  homepage "https://github.com/upstash/context7"
+  url "https://registry.npmjs.org/@upstash/context7-mcp/-/context7-mcp-1.0.17.tgz"
+  sha256 "7c9e67f808d8ab84d291a6b4cc5be192a9c68ef2f12485ab6f21cebe073d65a5"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *std_npm_args
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    json = <<~JSON
+      {"jsonrpc":"2.0","id":1,"method":"initialize","params":{"protocolVersion":"2025-03-26","capabilities":{},"clientInfo":{"name":"brew-test","version":"1.0"}}}
+    JSON
+    output = pipe_output("#{bin}/#{name}", json)
+    json_response = output.lines.find { |line| line.start_with?("{") }
+    parsed = JSON.parse(json_response)
+    assert_equal "Context7", parsed["result"]["serverInfo"]["name"]
+    assert_equal version, parsed["result"]["serverInfo"]["version"]
+    assert parsed["result"]["capabilities"]["tools"]["listChanged"]
```

Comment:
what is asserted here?

Reply:
Mostly that the answer follow the [MCP specs](https://modelcontextprotocol.io/specification/2025-06-18/server/tools#capabilities) (and that is it working by the way).
The actual output looks like this:
```json
{"result":{"protocolVersion":"2025-03-26","capabilities":{"tools":{"listChanged":true}},"serverInfo":{"name":"Context7","version":"1.0.17"},"instructions":"Use this server to retrieve up-to-date documentation and code examples for any library."},"jsonrpc":"2.0","id":1}
```
I can test the method `tools/list` instead of `initialize` if you prefer, it would return the list of tools provided by the MCP server.

Reply:
yeah, that is what I did with my mcp PRs, I think `tools/list` is better than just `initialize`, you can check `Formula/t/terraform-mcp-server.rb`

Reply:
i modified the test to assert the presence of the 2 tools names